project_task_parent_due_auto/tests/test_parent_due_auto.py

```
@@ -1,47 +1,36 @@
-                {
-                    "name": "Test Project",
-                    "privacy_visibility": "employees",
-                    "alias_name": "project+test",
-                }
-            )
+        cls.test_project = cls.env["project.project"].create(
+            {
+                "name": "Test Project",
+                "privacy_visibility": "employees",
+                "alias_name": "project+test",
+            }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)